codec_parameters_test.go

```diff
@@ -63,6 +63,9 @@ func TestCodecParameters(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 2, cp5.Channels())
 
+	err = cp5.SetExtraData([]byte{0, 0, 0, 1})
```

Comment:
Could you use `cp6` (at the bottom of the test function) instead to test the new methods and simplify with `require.NoError(t, cp6.SetExtraData([]byte{0, 0, 0, 1}))`